mne/viz/epochs.py

```diff
@@ -442,6 +442,194 @@ def plot_epochs(epochs, epoch_idx=None, picks=None, scalings=None,
     return fig
 
 
+def plot_epochs_concat(epochs, picks=None, scalings=None, n_epochs=8,
+                       n_channels=10, title_str='#%003i', show=True,
```

Comment:
same here, if n_chan < 10 you should still make 10 slots but only use the top n_chan